custom_components/sector/event.py

```diff
@@ -96,6 +152,11 @@ def _trigger_event(self, event_type, event_attributes):
     @property
     def state(self) -> str:
         """Return the latest event type for the device."""
+        _LOGGER.debug(
+            "SECTOR_EVENT: Returning state for device %s: %s",
+            self._serial_no,
+            self._last_event_type or "No events",
+        )
         return self._last_event_type or "No events"
```

Comment:
state should return the last event time, now it's returning the event type (so it's no longer an event entity).
We need to overwrite `__last_event_triggered` in the handling as otherwise, it won't be the correct time.